package.json

```
@@ -12,23 +12,23 @@
     "pino.d.ts",
     "bin.js",
     "browser.js",
-    "pretty.js",
-    "usage.txt",
     "test",
     "docs",
-    "example.js",
+    "smoke.js",
     "lib"
   ],
   "scripts": {
     "docs": "docsify serve",
     "browser-test": "airtap --local 8080 test/browser*test.js",
     "lint": "eslint .",
+    "smoker": "node ./node_modules/.bin/smoker test-ci-smoke",
```

```suggestion
    "smoker": "smoker test-ci-smoke",
```

this should work too